org.mozilla.Thunderbird.appdata.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <application>
-    <id type="desktop">org.mozilla.thunderbird</id>
+    <id type="desktop">org.mozilla.Thunderbird.desktop</id>
```

Comment:
Adding `.desktop` is not recommended according to the AppStream spec. See https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#sect-Metadata-GenericComponent

Reply:
Oh, my bad!

Reply:
Whilst it may not be recommended in the appstream spec it is currently required to actually build in flatpak due to a bug (fixed in 0.10.4 and 0.11.1 but not widely deployed yet). I made this change yesterday specifically to fix the build. I'm very happy to delay shipping this until after we have this fixed everywhere - but if you want to test a build you'll need to add the .desktop suffix for now.

Also, I think you'll need the capitalization to be exactly the same across everything. If upstream wants lowercase t in their thunderbird then we should change that everywhere whilst we still can.

Reply:
@nedrichards should I change it back then? Or are we going to wait?

Reply:
Well, you either have to change it to org.mozilla.Thunderbird or change the capitalization everywhere else.

As for the .desktop bit. I'm happy to wait - it looks like we might get builder upgrades more quickly than I feared - but until then your build will fail in validation.

Reply:
I'm personally fine with the capitalization. Seeing that you're happy to wait on the `.desktop` bit, I'll wait as well.